japicmp/src/main/java/japicmp/util/JApiClassFileFormatVersionHelper.java

```diff
@@ -0,0 +1,71 @@
+package japicmp.util;
+
+import japicmp.model.JApiClassFileFormatVersion;
+
+public class JApiClassFileFormatVersionHelper {
+
+	private JApiClassFileFormatVersionHelper() {
+	}
+
+	public static String getOldJdkVersion(JApiClassFileFormatVersion x) {
+		return getJdkVersion(x.getMajorVersionOld(), x.getMinorVersionOld());
+	}
+
+	public static String getNewJdkVersion(JApiClassFileFormatVersion x) {
+		return getJdkVersion(x.getMajorVersionNew(), x.getMinorVersionNew());
+	}
+
+	private static String getJdkVersion(int major, int minor) {
+		if (major == -1 || minor == -1) {
+			return "";
+		}
+		if (major == 45) {
+			return minor < 3 ? "JDK 1.0" : "JDK 1.1";
+		} else if (major == 46) {
+			return "JDK 1.2";
+		} else if (major == 47) {
+			return "JDK 1.3";
+		} else if (major == 48) {
+			return "JDK 1.4";
+		} else if (major == 49) {
+			return "JDK 5.0";
+		} else if (major == 50) {
+			return "JDK 6";
```

Comment:
What about `return "JDK " + (major - 44)`?

Reply:
Sounds good. I just refactored it here: https://github.com/siom79/japicmp/pull/405/commits/c4659f1b9f89a95d297a7b3df752cb5ca9ebd03b